civiwiki/api/models/article.py

```diff
@@ -2,6 +2,15 @@
 from django.db import models
 
 # Create your models here.
+class ArticleManager(models.Manager):
+   def serialize(self, account, filter=None):
+      data ={
+         "category": Topic.category,
+         "topic": Topic.topic,
+         "bill": Topic.bill,
+         "all_objects": Topic.objects,
+      }
+      return json.dumps(data)
 class Topic(models.Model):
     '''
```

Comment:
you also need to make sure we can access these new functions you have created! you need to change the objects value from **models.Manager()**, to **TopicManager()** _Currently labeled ArticleManager()_. If you look at the instantiation of **TopicManager** you will see that we actually inherit all of the classes from **models.Manager()**, all we are doing is creating a new manager class with a couple of extra functions.